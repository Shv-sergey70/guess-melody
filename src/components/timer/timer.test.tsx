import * as React from 'react';
import {configure, shallow} from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';
import Timer from "./timer";

configure({adapter: new Adapter()});

describe(`Timer renders correctly`, () => {
  test(`Timer has a lot of time - usual behaviour`, () => {
    const tree = shallow(
        <Timer
          minutes={`02`}
          seconds={`52`}
          isFinishing={false}
        />
    );

    expect(tree).toMatchSnapshot();
  });

  test(`Timer has low time - should be added finishing class`, () => {
    const tree = shallow(
        <Timer
          minutes={`00`}
          seconds={`18`}
          isFinishing={true}
        />
    );

    expect(tree).toMatchSnapshot();
  });
});
