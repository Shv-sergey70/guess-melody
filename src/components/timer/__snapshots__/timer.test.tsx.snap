// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Timer renders correctly Timer has a lot of time - usual behaviour 1`] = `ShallowWrapper {}`;

exports[`Timer renders correctly Timer has low time - should be added finishing class 1`] = `ShallowWrapper {}`;
